### PR TITLE
Add subset tag and synonym type to UBERON:8410010

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1665

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
Hi @dosumis ,
I added the HCA subset tag, and also synonym type property = plural to 2 existing synonyms while I was there. 
Now ready for your review please.

File: uberon_edit.obo [hunk 1]
```diff
@@ -195254,12 +195254,13 @@ id: UBERON:8410010
...
 synonym: "fimbria of fallopian tube" EXACT []
-synonym: "fimbriae of fallopian tube" EXACT []
-synonym: "fimbriae of uterine tube" EXACT []
+synonym: "fimbriae of fallopian tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
+synonym: "fimbriae of uterine tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
 xref: FMA:18308
 xref: http://orcid.org/0000-0001-7505-5418
 is_a: UBERON:0013515 ! subdivision of oviduct
 relationship: part_of UBERON:0003889 ! fallopian tube
+property_value: added_for_HCA "" xsd:string
```
Comment (dosumis) [inline uberon_edit.obo:195263]: See comment on inguinal skin region.

File: uberon_edit.obo [hunk 2]
```diff
@@ -195254,12 +195254,13 @@ id: UBERON:8410010
 name: fimbria of uterine tube
 def: "A fringe of tissue around the ostium of the fallopian tube, in the direction of the ovary." [WikipediaVersioned:Fallopian_tube&oldid=998395727]
 synonym: "fimbria of fallopian tube" EXACT []
-synonym: "fimbriae of fallopian tube" EXACT []
-synonym: "fimbriae of uterine tube" EXACT []
+synonym: "fimbriae of fallopian tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
+synonym: "fimbriae of uterine tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
```
Comment (dosumis) [inline uberon_edit.obo:195258]: Looks like some error in how this was added.

Should be like this.
image

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments

**Comment by paolaroncaglia** [comment]
@dosumis 
I fixed the subset tag, but I can't get the synonym type (plural) to be displayed as you suggested. I need to get through more pull request reviews now, so I left the synonyms as exact, and I'll make a note to look into this again at another time please.

**Review by dosumis — APPROVED** [review]

**Pull request merged**